library/common/build.gradle.kts

```diff
@@ -10,8 +10,10 @@ animealPlugin {
 }
 
 dependencies {
+    implementation(projects.library.foundation)
     implementation(libs.androidx.datastore)
     implementation(libs.hilt.android)
     implementation(libs.androidx.viewmodel)
+    implementation(project(mapOf("path" to ":library:foundation")))
```

Comment:
What is the diff between this and #13 line?

Reply:
My bad. Removed.